.github/workflows/spanner-emulator-system-tests.yaml

```diff
@@ -39,10 +39,12 @@ jobs:
         with:
           php-version: '7.4'
           tools: pecl
-          extensions: bcmath, grpc-1.37.1
+          extensions: bcmath, grpc
 
       - name: Install dependencies
         run: |
+          composer config minimum-stability dev -d Spanner/
+          composer config repositories.local  --json '{"type":"path", "url": "../Core", "options": {"versions": {"google/cloud-core": "1.45"}}}' -d Spanner
```

Comment:
This is because we have an additional dev dependency in spanner for `google/cloud-pubsub`, and in order for composer to be smart enough to resolve `cloud-core` for that dependency as well, we need to do the slightly more special way here, where we set the "versions" option.